code/datums/gamemode/role/plague_mouse.dm

```diff
@@ -0,0 +1,25 @@
+
+/datum/role/plague_mouse
+	name = PLAGUEMOUSE
+	id = PLAGUEMOUSE
+	special_role = PLAGUEMOUSE
+	required_pref = ROLE_MINOR
```

Comment:
I suggest you also add `wikiroute = ROLE_MINOR`

Reply:
done